pipelines/cli.go

```diff
@@ -0,0 +1,79 @@
+package pipelines
+
+import (
+	corecommon "github.com/jfrog/jfrog-cli-core/v2/docs/common"
+	"github.com/jfrog/jfrog-cli/docs/common"
+	"github.com/jfrog/jfrog-cli/docs/pipelines/status"
+	"github.com/jfrog/jfrog-cli/docs/pipelines/sync"
+	"github.com/jfrog/jfrog-cli/docs/pipelines/syncstatus"
+	"github.com/jfrog/jfrog-cli/docs/pipelines/trigger"
+	"github.com/jfrog/jfrog-cli/docs/pipelines/version"
+	"github.com/jfrog/jfrog-cli/utils/cliutils"
+	"github.com/urfave/cli"
+)
+
+func GetCommands() []cli.Command {
+	return cliutils.GetSortedCommands(cli.CommandsByName{
+		{
+			Name:         "status",
+			Flags:        cliutils.GetCommandFlags(cliutils.Status),
+			Aliases:      []string{"s"},
+			Usage:        status.GetDescription(),
+			HelpName:     corecommon.CreateUsage("pl status", status.GetDescription(), status.Usage),
+			ArgsUsage:    common.CreateEnvVars(),
+			BashComplete: corecommon.CreateBashCompletionFunc(),
+			Action: func(c *cli.Context) error {
+				return fetchLatestPipelineRunStatus(c)
+			},
+		},
+		{
+			Name:         "trigger",
+			Flags:        cliutils.GetCommandFlags(cliutils.Trigger),
+			Aliases:      []string{"t"},
+			Usage:        trigger.GetDescription(),
+			HelpName:     corecommon.CreateUsage("pl trigger", trigger.GetDescription(), trigger.Usage),
+			UsageText:    trigger.GetArguments(),
+			ArgsUsage:    common.CreateEnvVars(),
+			BashComplete: corecommon.CreateBashCompletionFunc(),
+			Action: func(c *cli.Context) error {
+				return triggerNewRun(c)
+			},
+		},
+		{
+			Name:         "version",
+			Flags:        cliutils.GetCommandFlags(cliutils.Version),
+			Aliases:      []string{"v"},
+			Usage:        version.GetDescription(),
+			HelpName:     corecommon.CreateUsage("pl version", version.GetDescription(), version.Usage),
+			BashComplete: corecommon.CreateBashCompletionFunc(),
+			Action: func(c *cli.Context) error {
+				return getVersion(c)
+			},
+		},
+		{
+			Name:         "sync",
+			Flags:        cliutils.GetCommandFlags(cliutils.Sync),
+			Aliases:      []string{"sy"},
+			Usage:        sync.GetDescription(),
+			HelpName:     corecommon.CreateUsage("pl sync", sync.GetDescription(), sync.Usage),
+			UsageText:    sync.GetArguments(),
+			ArgsUsage:    common.CreateEnvVars(),
+			BashComplete: corecommon.CreateBashCompletionFunc(),
+			Action: func(c *cli.Context) error {
+				return syncPipelineResources(c)
+			},
+		},
+		{
+			Name:         "syncstatus",
+			Flags:        cliutils.GetCommandFlags(cliutils.SyncStatus),
+			Aliases:      []string{"ss"},
+			Usage:        syncstatus.GetDescription(),
+			HelpName:     corecommon.CreateUsage("pl syncstatus", syncstatus.GetDescription(), syncstatus.Usage),
+			ArgsUsage:    common.CreateEnvVars(),
+			BashComplete: corecommon.CreateBashCompletionFunc(),
+			Action: func(c *cli.Context) error {
+				return getSyncPipelineResourcesStatus(c)
+			},
+		},
+	})
+}
```

Comment:
Let's move the code of the following files into this file. There's no need to keep ihem on sperate files.
<img width="195" alt="image" src="https://user-images.githubusercontent.com/7830056/214617708-79e15e8d-e942-4cc7-a988-c4ff2e82c294.png">